setup.py

```diff
@@ -60,8 +60,7 @@
 ]
 
 install_requires = [
-    "invenio-records-resources>=0.19.4,<0.20.0",
-    "invenio-i18n>=1.3.1",
```

Comment:
comes from invenio-records-resources, duplicating causes version conflicts and we can actually rely on what comes from records-resources